nss-day-cohort-50/nashville-kennels-holmes
# user can search for animals/employees/locations

#### Changes Made
1. Modified file `SearchResults.js` to include `search function`.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout sb-display-search-results
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > Make sure search functionality works
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
